packages/api-core/src/graphql/schema.graphql

```diff
@@ -34,6 +34,100 @@ enum MassUnit {
   oz
 }
 
+"Relational Operator Types used in filtering inside a single condition"
+enum RelationalOperatorTypes{
+  "Begins With used with String types"
+  beginsWith
+
+  "Ends With used with String types"
+  endsWith
+
+  "Equal"
+  eq
+
+  "Greater Than"
+  gt
+
+  "Greater Than or Equal"
+  gte
+
+  "In"
+  in
+
+  "Less Than"
+  lt
+
+  "Less Than or Equal"
+  lte
+
+  "Not Equal"
+  ne
+
+  "Not In"
+  nin
+
+  "Regex"
+  regex
+}
+
+"Single Condition for filterSearch, use exactly one of the optional input value type"
+input SingleConditionInput {
+  "Value to filter if it is Boolean input"
+  boolValue: Boolean
+
+  "Flag to set if the regex is case insensitive"
+  caseSensitive: Boolean
+
+  "Value to filter if it is Date input"
+  dateValue: DateTime
+
+  "Value to filter if it is Float Array input" 
+  floatArrayValue: [Float]
+
+  "Value to filter if it is Float input"
+  floatValue: Float
+
+  "Value to filter if it is Int Array input" 
+  intArrayValue: [Int]
+
+  "Value to filter if it is Int input"
+  intValue: Int
+
+  "Field name"
+	key : String!
+
+  "Logical NOT operator to negate the condition"
+  logicalNOT: Boolean
```

Comment:
I can't think of any reason why you would be capitalize all of NOT

Reply:
Updated to logicalNot